pages/form-builder/responses/[[...params]].tsx

```diff
@@ -0,0 +1,221 @@
+import React, { ReactElement, useState } from "react";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useTranslation } from "next-i18next";
+import { getFullTemplateByID } from "@lib/templates";
+import { unstable_getServerSession } from "next-auth";
+import { authOptions } from "@pages/api/auth/[...nextauth]";
+import { AccessControlError, createAbility } from "@lib/privileges";
+import { NextPageWithLayout } from "@pages/_app";
+import { PageTemplate, Template } from "@components/form-builder/app";
+import { Button, useDialogRef, Dialog } from "@components/form-builder/app/shared";
+import { StyledLink } from "@components/globals/StyledLink/StyledLink";
+import { GetServerSideProps } from "next";
+import { FormRecord, VaultSubmissionList } from "@lib/types";
+import { listAllSubmissions } from "@lib/vault";
+import { logMessage } from "@lib/logger";
+import { useSession } from "next-auth/react";
+
+interface ResponsesProps {
+  vaultSubmissions: VaultSubmissionList[];
+}
+
+const Responses: NextPageWithLayout<ResponsesProps> = ({ vaultSubmissions }: ResponsesProps) => {
+  const { t } = useTranslation("form-builder");
+
+  const { status } = useSession();
+  const secondaryButtonClass =
+    "whitespace-nowrap relative py-0 px-2 rounded-lg border-2 border-solid inline-flex items-center active:top-0.5 focus:outline-[3px] focus:outline-blue-focus focus:outline focus:outline-offset-2 focus:bg-blue-focus focus:text-white-default disabled:cursor-not-allowed disabled:text-gray-500  bg-white-default text-black-default border-black-default hover:text-white-default hover:bg-gray-600 active:text-white-default active:bg-gray-500";
+
+  const dialogConfirmReceipt = useDialogRef();
+  const [isShowConfirmReceiptDialog, setIsShowConfirmReceiptDialog] = useState(false);
+  const dialogConfirmReceiptHandleClose = () => {
+    setIsShowConfirmReceiptDialog(false);
+    dialogConfirmReceipt.current?.close();
+  };
+  const buttonActionsConfirmReceipt = (
+    <Button onClick={dialogConfirmReceiptHandleClose}>{t("responses.confirmReceipt")}</Button>
```

Comment:
Not sure if the style you looking for here lines up but you might be able to pass a theme prop to the button

```
theme?: "primary" | "secondary" | "destructive" | "link" | "icon";
```

Reply:
As I mentionned in the PR the styling of the modals was out of scope.  I just grabbed what Pete has started and all that styling will get sorted in the next PR.

Reply:
Yeah, I have yet to clean up all the styling going on in this page.  
Will get to it before I convert to ready for review.

Reply:
Done. I tried to re-use the Form-builder buttons (and other components) as much as possible. One nav item is a link, that is styled to look like a button